modules/ROOT/pages/_partials/sync-api/sync-function-api-access.adoc

```diff
@@ -58,5 +57,6 @@ access ("snej", null);
 <.> The null arguments mean these are treated as no-ops
 ====
 
+WARNING: Invoking the 'access()' function multiple times to grant the same user access to the same channel will result in negative performance implications, such as large fetches or request timeouts.
```

Comment:
```suggestion
WARNING: If you invoke the `access()` function multiple times to grant the same user access to the same channel, you could see negative performance effects, such as large fetches or request timeouts. 
```

I don't think this properly used the ` that we use for monospace? And I think we can tighten up the language a bit. :) 

Remember, it's okay to address the user directly! It makes things more active and impactful.

Reply:
Done! I get them mixed up all time time still 😅